src/main/java/org/opensearch/sdk/ExtensionSettings.java

```diff
@@ -93,14 +93,23 @@ private ExtensionSettings() {
      * Instantiate this class using the specified parameters.
      *
      * @param extensionName  The extension name. Provided to OpenSearch as a response to initialization query. Must match the defined extension name in OpenSearch.
+     * @param shortName  The shortened name for the extension
      * @param hostAddress  The IP Address to bind this extension to.
      * @param hostPort  The port to bind this extension to.
      * @param opensearchAddress  The IP Address on which OpenSearch is running.
      * @param opensearchPort  The port on which OpenSearch is running.
      */
-    public ExtensionSettings(String extensionName, String hostAddress, String hostPort, String opensearchAddress, String opensearchPort) {
+    public ExtensionSettings(
+        String extensionName,
+        String shortName,
```

Comment:
```suggestion
        String shortExtensionName,
```

Reply:
I'll update it, I was torn between different names for this variable. One of the naming ideas was `extensionAbbr`, but I didn't know if was appropriate to use an abbreviation in a variable name signifying that this variable is an abbreviation. It felt like it was a crossword puzzle clue.

Reply:
Updated